packages/jest-types/src/Config.ts

```diff
@@ -90,15 +90,14 @@ export type DefaultOptions = {
   slowTestThreshold: number;
   snapshotSerializers: Array<Path>;
   testEnvironment: string;
-  testEnvironmentOptions: Record<string, unknown>;
+  testEnvironmentOptions?: Record<string, any>;
```

Comment:
please keep it as `unknown`

Reply:
and why optional?

Reply:
ok man i was following this  https://github.com/facebook/jest/issues/7185#issuecomment-474978713 pattern specified here , will make immediate change